src/server/stream_family_test.cc

```diff
@@ -725,4 +725,13 @@ TEST_F(StreamFamilyTest, XAck) {
   resp = Run({"xreadgroup", "group", "cgroup", "consumer", "streams", "foo", "0"});
   EXPECT_THAT(resp, ArrLen(0));
 }
+
+TEST_F(StreamFamilyTest, XInfo) {
+  Run({"xgroup", "create", "foo", "cgroup", "0", "mkstream"});
+  auto resp = Run({"xinfo", "groups", "foo"});
+  EXPECT_THAT(resp.GetVec(), ElementsAre("name", "cgroup", "consumers", IntArg(0), "pending",
```

Comment:
Same here, please use `EXPECT_THAT(resp, RespArray(ElementsAre(...)))`
It has the advantage of not crashing if the result is not an array (that's why in many cases we used to add a check that the result is an array)

Reply:
fixed